Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM registry.fedoraproject.org/fedora:28
-ADD build.sh /root
-RUN mkdir /root/src
-COPY Makefile coreos-* /root/src/
-RUN ./root/build.sh && rm -f /root/build.sh # cache20180523
\ No newline at end of file
+WORKDIR /root/src
```

Comment:
We need to install `git` before running `build.sh`

Reply:
Ah yeah, thanks for testing this.  It worked for me locally because I'd already done the recursive clone.